packages/serverless-orchestration/src/ServerlessHub.js

```diff
@@ -50,7 +50,12 @@ const defaultHubConfig = {
   rejectSpokeDelay: 120, // 2 min.
 };
 
+const waitForLoggerDelay = process.env.WAIT_FOR_LOGGER_DELAY || 5;
+
 hub.post("/", async (req, res) => {
+  // Use a custom logger if provided. Otherwise, initialize a local logger.
+  // Note: no reason to put this into the try-catch since a logger is required to throw the error.
+  const logger = customLogger || createNewLogger();
```

Comment:
this is taken from https://github.com/UMAprotocol/protocol/pull/3686. will rebase once that PR is merged in